fivetran_provider_async/hooks.py

```diff
@@ -92,7 +95,7 @@ async def get_connector_async(self, connector_id):
         resp = await self._do_api_call_async(("GET", endpoint))
         return resp["data"]
 
-    async def get_sync_status_async(self, connector_id, previous_completed_at):
+    async def get_sync_status_async(self, connector_id, previous_completed_at, reschedule_time=0):
```

Comment:
nitpick: should we add type hint to this hook?

Reply:
+1.

Reply:
https://github.com/astronomer/airflow-provider-fivetran-async/pull/25/commits/16b919341e5c476ac73a27ff5e09f90babd495ea